packages/interbit-ui-tools/src/middleware/sagas/index.js

```diff
@@ -2,24 +2,25 @@ const { put, takeEvery, call } = require('redux-saga/effects')
 
 const { actionTypes, actionCreators } = require('../actions')
 const getHtmlConfig = require('../getConfigFromStaticHtml')
-const {
-  getConfiguredChains,
-  getConfiguredPeers,
-  entireTree
-} = require('../selectors')
 const { LOG_PREFIX } = require('../constants')
 const { connectToPeers } = require('./connections')
 const { interbitContext } = require('./interbit')
-const { loadPrivateChain, loadChain, sponsorChain } = require('./chains')
+const { loadStaticChains, loadPrivateChain, sponsorChain } = require('./chains')
 
 function* rootSaga() {
   console.log(`${LOG_PREFIX}: *rootSaga(): loading interbit`)
   yield call(loadInterbitSaga)
 
-  console.log(
-    `${LOG_PREFIX}: *rootSaga(): watching for ${actionTypes.PRIVATE_CHAIN_SAGA}`
-  )
+  console.log(`${LOG_PREFIX}: *rootSaga(): listening for: `, [
+    actionTypes.STATIC_CHAINS_SAGA,
+    actionTypes.PRIVATE_CHAIN_SAGA,
+    actionTypes.SPONSOR_CHAIN_SAGA
+  ])
+  yield takeEvery(actionTypes.STATIC_CHAINS_SAGA, staticChainsSaga)
   yield takeEvery(actionTypes.PRIVATE_CHAIN_SAGA, privateChainSaga)
+  yield takeEvery(actionTypes.SPONSOR_CHAIN_SAGA, sponsorChainSaga)
+
+  yield put(actionCreators.staticChainsSaga())
 }
```

Comment:
This starts the saga to load the static chains. We know that the preconditions are ready.